src/yuzu/configuration/configure_system.cpp

```diff
@@ -34,6 +46,19 @@ ConfigureSystem::ConfigureSystem(Core::System& system_, QWidget* parent)
         }
     });
 
+    const auto locale_check = [this](int index) {
+        bool validLocale = CHECK_LOCALE(ui->combo_region->currentIndex(), ui->combo_language->currentIndex());
+        ui->label_warn_invalid_locale->setVisible(!validLocale);
+        if (!validLocale) {
+            ui->label_warn_invalid_locale->setText(
+                QStringLiteral("Warning: \"%1\" is not a valid language for region \"%2\"")
```

Comment:
```suggestion
                tr("Warning: \"%1\" is not a valid language for region \"%2\"")
```

Since this should be translatable